_posts/2019-08-06-pre-post-events-sustainable-trains.md

```diff
@@ -0,0 +1,114 @@
+---
+layout: post
+title: "Pre-/Post-Events: sustainable train travels from and to RustFest Barcelona"
+authors:
+  - dns2utf8
+socialImageSrc: "/assets/posts/barcelona/cfp-opens.png"
+---
+
+## The big plan 🚆
+
+What if you could meet with other Rustaceans while you travel to and from RustFest Barcelona?
```

Comment:
I'd be a bit more wordy here and say what you want to achieve.

```suggestion
What if you could meet with other Rustaceans while you travel to and from RustFest Barcelona?
We want to make that happen and are organizing a community travel from the north of Europe all the way down to Barcelona, with stops in between.
We want to encourage more people to join us or plan similar travel events.
```

something like that.